docs/ml-tuning.md

```diff
@@ -62,7 +62,7 @@ To help construct the parameter grid, users can use the [`ParamGridBuilder`](api
 
 After identifying the best `ParamMap`, `CrossValidator` finally re-fits the `Estimator` using the best `ParamMap` and the entire dataset.
 
-## Example: model selection via cross-validation
+## Examples: model selection via cross-validation
```

Comment:
I hesitated about where to modify this place. For there are not may links in http://spark.apache.org/docs/latest/ml-tuning.html like those in featureAlgos http://spark.apache.org/docs/latest/ml-features.html.
But now, I tend to use `**examples...**` for standardization.